assets/dev/js/editor/elements/views/container.js

```diff
@@ -181,16 +181,33 @@ const ContainerView = BaseElementView.extend( {
 	},
 
 	/**
-	 * Add a `Save as Template` button to the context menu.
+	 * Insert a new container inside an existing container.
+	 *
+	 * @returns {void}
+	 */
+	addNewContainer() {
+		const containerSelector = ( false === this.getContainer().parent.parent ) ? this.getContainer() : this.getContainer().parent;
+
+		$e.run( 'document/elements/create', {
+			model: {
+				elType: 'container',
+			},
+			container: containerSelector,
+		} );
```

Comment:
Preferably I would like @iNewLegend to double check part for me. 

In his work for the columns widget, he added: 
			options: {
				at: this.$el.index() + 1,
			},
See: https://github.com/elementor/elementor/blob/02fb2b7e3388403d44d3d3ec1557a80e364c0312/assets/dev/js/editor/elements/views/column.js#L194. 

I am not sure if we need that as well. I am not familiar with these document element methods yet. 
I believe it will be good if Leonid checks it. He seems to better versed in this area.

Reply:
It was there to place the column in the right position ( right after current column ).